docs/cog_guides/audio.rst

```diff
@@ -0,0 +1,3739 @@
+
+.. code-block:: sh
+
+	curl -LOz Lavalink.jar https://github.com/Cog-Creators/Lavalink-Jars/releases/latest/download/Lavalink.jar
+
+Run the following, replacing ``<Lavalink path>`` with the Lavalink path you generated earlier (``<datapath>/cogs/Audio``).
+
+.. code-block:: sh
+
+	chmod a+rx <Lavalink path>/auto_update.sh
+
+Now we need to create a service file so that systemd can do it's magic. Run the following command:
+
+.. code-block:: sh
+
+	sudo -e /etc/systemd/system/lavalink.service
+	
+Next, paste in the example below, but replacing the following:
+
+* ``<Jar executable path>`` - You can find your Java path by running ``which java``.
+* ``<Lavalink path>`` - The parent folder where your Lavalink executable can be located (usually in ``<datapath>/cogs/Audio``).
+* ``<username>`` - Your username on the host machine (run ``echo $USER``).
+
+.. code-block:: ini
+
+	[Unit]  
+	Description=lavalink  
+	After=multi-user.target  
+
+	[Service]
+	ExecStart=<Java executable path> -Djdk.tls.client.protocols=TLSv1.2 -jar < Lavalink path >/Lavalink.jar
+	WorkingDirectory=<Lavalink path>
+	User=<username>
+	Group=<username>
+	ExecStartPre=/bin/bash <Lavalink path>/auto_update.sh # Comment this line out if you did not create the auto_update.sh
+	Type=idle
+	Restart=always
+	RestartSec=15
+
+	[Install]
+	WantedBy=multi-user.target
+
+Finally, we need to start and enable the service. Run the following commands, separately.
+
+.. code-block:: sh
+	
+	sudo systemctl start lavalink
+	sudo systemctl enable lavalink
+
+These commands always need to be ran when starting the lavalink service to ensure that the
+service runs in the background.
+
+Finally, you can run the following to retrieve logs for the service, when you need them:
+
+.. code-block:: sh
+	
+	sudo journalctl -u lavalink
+
+.. _obtaining-the-latest-lavalink:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Obtaining the latest Lavalink.jar on a Red update
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**To manually download the jar**
+
+1. Stop your bot.
+2. Click `here <https://github.com/Cog-Creators/Lavalink-Jars/releases/latest>`__.
+3. Download the Lavalink.jar file.
+
+**If you are not running your own instance of Lavalink**
+
+1. Stop your bot by running ``[p]shutdown``.
+2. Move the Lavalink.jar to your ``<datapath>/cogs/Audio/`` folder.
+3. Start your bot.
+
+**If you are running your own instance of Lavalink**
+
+1. Stop all instances of Red running on the host.
+2. Stop the Lavalink process.
+3. Move the Lavalink.jar to where your old Lavalink.jar is located and overwrite the old file.
+4. Start up the new Lavalink.jar process via a process manager like systemd, or by running:
+
+   .. code-block:: sh
+   
+	   java -jar Lavalink.jar -Djdk.tls.client.protocols=TLSv1.2
+
+5. Start up your bots
+
+.. _remote-lavalink:
+
+---------------------------------------------------------------
+Setting up an unmanaged Lavalink node on a remote VPS or server
```

Comment:
It's a tough one - the documentation of running a local unmanaged and a remote unmanaged is mostly the same with a few key differences.


local unmanaged, red would have installed java for you so the docs should tell you to use java 11. 
remote unmanaged is wholly isolated from Red - in this case, Java 13 and 18 should be the goto java versions, the applications YAML will also differ as the local managed the binding IP should be localhost, while a remote unmanaged will be 0.0.0.0 as it will have to communicate over the internet.

IMHO it makes sense and it's clear for a user to have these 2 sections separate - that being said, remote unmanaged is obviously outside the scope of Red docs

Reply:
It was not clear to me that one is talking about exclusively remote hosting, and the other only local hosting. If that's the difference, these sections should make it more clear by making something akin to that be the header of each, rather than just "advanced usage".